src/main.js

```diff
@@ -338,17 +338,29 @@ class MiniGraphCard extends LitElement {
 
   renderLegend() {
     if (this.visibleLegends.length <= 1 || !this.config.show.legend) return;
+
+
     return html`
       <div class="graph__legend">
-        ${this.visibleLegends.map(entity => html`
-          <div class="graph__legend__item"
-            @click=${e => this.handlePopup(e, this.entity[entity.index])}
-            @mouseenter=${() => this.setTooltip(entity.index, -1, this.getEntityState(entity.index), 'Current')}
-            @mouseleave=${() => (this.tooltip = {})}>
-            ${this.renderIndicator(this.entity[entity.index].state, entity.index)}
-            <span class="ellipsis">${this.computeName(entity.index)}</span>
-          </div>
-        `)}
+        ${this.visibleLegends.map((entity) => {
+    let legend = this.computeName(entity.index);
+    const state = this.getEntityState(entity.index);
+
+    const { show_legend_state } = this.config.entities[entity.index];
+    if (show_legend_state) {
+      legend += ` (${this.computeState(state)}${this.computeUom(entity.index)})`;
```

Comment:
Can you put the state in an additional `span` with a suitable class? (If there already are suitable classes for state and UOM, you could maybe reuse them.)

I think there is a CSS property then to add the braces to the content. 

That way users could use `card-mod` to style the appearance of the state as they wish.

Please make sure the defaults look good in different situations, e.g. long and short state names.

Reply:
> put the state in an additional span

Absolutely correct.

Also, please think about an order: in some cases a user may want to have UoM before a value.

Reply:
I think, a separate `span` should do the trick for that as well, together with the CSS `direction` property.